src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -478,17 +500,26 @@ def main():
     conf = None
 
     try:
+        # setup container.conf file for the callback server logs
+        if click.confirm(f"Set up a containers.conf file at {loader_common_names.CONTAINERS_CONF_PATH} if does not exist already?\n"
```

Comment:
You should check if the config requires modification and then ask, rather than asking every single run

Reply:
Updated!

Reply:
:+1: